### PR TITLE
Prep first release at 0.0.2

### PR DESCRIPTION
**Pull request by backus**
- I used 0.0.1 to squat rubygems so I can't overwrite that
 - I removed attribute to mutant in the README as Markus asked
 - I updated the author and links to the project
 - I updated the description in the gemspec which was a bit stale.

### CODE REVIEW

File: mutest-rspec.gemspec [hunk 1]
```diff
@@ -3,11 +3,11 @@ require File.expand_path('../lib/mutest/version', __FILE__)
 Gem::Specification.new do |gem|
   gem.name        = 'mutest-rspec'
   gem.version     = Mutest::VERSION.dup
-  gem.authors     = ['Markus Schirp']
-  gem.email       = ['mbj@schirp-dso.com']
+  gem.authors     = ['Daniel Gollahon', 'John Backus']
+  gem.email       = ['johncbackus@gmail.com', 'daniel.gollahon@gmail.com']
```
Comment (dgollahon) [inline mutest-rspec.gemspec]: should these be in the same order as the names?

Comment (dgollahon) [inline mutest-rspec.gemspec]: or does it matter?

**Comment by dgollahon** [comment]
We should probably add changelog entries for the new mutations.

**Comment by backus** [comment]
> We should probably add changelog entries for the new mutations.

I think we should save it for our first minor release. This release is basically so we can start using it for work

File: mutest-rspec.gemspec (hunk 1 above)
Comment (backus) [inline mutest-rspec.gemspec]: Order doesn't really matter

**Comment by dgollahon** [comment]
> I think we should save it for our first minor release.

Well, we should still show it in the between `0.0.1` and `0.0.2` section when we get around to it as the new mutations won't be part of a minor release.

**Comment by backus** [comment]
> Well, we should still show it in the between 0.0.1 and 0.0.2 section when we get around to it as the new mutations won't be part of a minor release.

:thumbsup:

File: mutest.gemspec [hunk 2]
```diff
@@ -4,10 +4,10 @@ require File.expand_path('../lib/mutest/version', __FILE__)
 Gem::Specification.new do |gem|
   gem.name        = 'mutest'
   gem.version     = Mutest::VERSION.dup
-  gem.authors     = ['John Backus']
-  gem.email       = ['johncbackus@gmail.com']
+  gem.authors     = ['Daniel Gollahon', 'John Backus']
+  gem.email       = ['johncbackus@gmail.com', 'daniel.gollahon@gmail.com']
   gem.description = 'Mutation testing for ruby'
-  gem.summary     = 'Mutation testing tool for ruby under MRI and Rubinius'
+  gem.summary     = 'Mutation testing tool for MRI'
```
Comment (dgollahon) [inline mutest.gemspec]: Since this isn't meaningfully different, we should probably make the description and summary the same.

Comment (backus) [inline mutest.gemspec]: Sure

**Comment by backus** [comment]
Updated summary. I also removed one rubinius conditional from the codebase since I also removed the RBX description for the gem

**Pull request merged**